### PR TITLE
[Windows] move llvm to toolset

### PR DESCRIPTION
**Pull request by mikhailkoliada**
# Description

Pinned latest llvm version using the recently added `Get-LatestChocoPackageVersion` function.
I also propose we dropping the `clang++ --version` check in tests as `clang++` calls are literally equal to the `clang -x` and `clang -lstdc++` ones (i.e. binary is in fact the same but installed a little bit differently).

## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-LLVM.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+################################################################################
+##  File:  Install-LLVM.ps1
+##  Desc:  Install the latest stable version of llvm and clang compilers
+################################################################################
+
+$llvmVersion = (Get-ToolsetContent).llvm.version
+$latestChocoVersion = Get-LatestChocoPackageVersion -TargetVersion ${llvmVersion} -PackageName "llvm"
+Choco-Install -PackageName llvm -ArgumentList "--version ${latestChocoVersion}"
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Install-LLVM.ps1:8]:
```suggestion
Choco-Install -PackageName llvm -ArgumentList "--version=${latestChocoVersion}"
```

**Review by shilovmaksim — DISMISSED** [review]

**Review by dmitry-shibanov — DISMISSED** [review]

File: images/win/scripts/Installers/Install-LLVM.ps1 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+################################################################################
+##  File:  Install-LLVM.ps1
+##  Desc:  Install the latest stable version of llvm and clang compilers
+################################################################################
+
+$llvmVersion = (Get-ToolsetContent).llvm.version
+$latestChocoVersion = Get-LatestChocoPackageVersion -TargetVersion ${llvmVersion} -PackageName "llvm"
```
Comment (al-cheb) [inline images/win/scripts/Installers/Install-LLVM.ps1:7]: nitpick: remove {} -> $llvmVersion

**Review by dibir-magomedsaygitov — DISMISSED** [review]

File: images/win/scripts/Installers/Install-LLVM.ps1 (hunk 2 above)
Comment (mikhailkoliada) [reply images/win/scripts/Installers/Install-LLVM.ps1]: @al-cheb done

**Review by shilovmaksim — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Comment by kamasawaddee** [comment]
> # Description
> 
> Pinned latest llvm version using the recently added `Get-LatestChocoPackageVersion` function.
> I also propose we dropping the `clang++ --version` check in tests as `clang++` calls are literally equal to the `clang -x` and `clang -lstdc++` ones (i.e. binary is in fact the same but installed a little bit differently).
> 
> #### Related issue: https://github.com/actions/virtual-environments-internal/issues/2918
> 
> ## Check list
> - [X] Related issue / work item is attached
> - [ ] Tests are written (if applicable)
> - [ ] Documentation is updated (if applicable)
> - [ ] Changes are tested and related VM images are successfully generated
> 

Exit

**Pull request merged**